### PR TITLE
Initiating abort should skip prerequisite checks

### PR DESCRIPTION
**Pull request by ramkrsna**
This fixes the abort behaviour, which was prompting for the pre requisite checks to run.

### CODE REVIEW

File: deploy/gk-deploy [hunk 1]
```diff
@@ -385,7 +386,7 @@ else
   exit 1
 fi
 
-if [[ ${ABORT} -eq 1 ]]; then
+if [[ ${ABORT} -eq 1 ]] && [[ ${SKIP_PREREQ} -eq 1]]; then
```
Comment (humblec) [inline deploy/gk-deploy]: "-eq 1]]" should be "-eq 1 ]]"

**Comment by humblec** [comment]
@MohamedAshiqrh @raghavendra-talur PTAL.

**Comment by ramkrsna** [comment]
@humblec sent in the commit with the required changed. @raghavendra-talur @MohamedAshiqrh PTAL

**Comment by humblec** [comment]
LGTM.

File: deploy/gk-deploy [hunk 2]
```diff
@@ -233,6 +233,7 @@ while [[ $# -ge 1 ]]; do
         ;;
         -abort)
         ABORT=1
+        SKIP_PREREQ=1
```
Comment (MohamedAshiqrh) [inline deploy/gk-deploy]: Why can't we add a abort check in SKIP_PREREQ?

Comment (MohamedAshiqrh) [inline deploy/gk-deploy]: I just think If we use SKIP_PREREQ for something else tomo then this will impact Abort as it always uses it.

Comment (MohamedAshiqrh) [inline deploy/gk-deploy]: [Here](https://github.com/gluster/gluster-kubernetes/blob/master/deploy/gk-deploy#L265)

Comment (humblec) [inline deploy/gk-deploy]: @MohamedAshiqrh the SKIP_PREREQ should be used for the same purpose even later. Also I feel the flag has to be enabled as soon as the option is provided. but agree on, yes there are different ways to fix it.

Comment (MohamedAshiqrh) [inline deploy/gk-deploy]: Agreed. LGTM. No other comments.

Comment (ramkrsna) [inline deploy/gk-deploy]: Here different solutions are possible . I'm explicitly handling only when the --abort is used. We could test the both the approaches

Comment (humblec) [inline deploy/gk-deploy]: @ramkrsna to make it simple, we can just add the check in as mentioned by @MohamedAshiqrh . We will revisit this change later if at all required.

**Comment by humblec** [comment]
LGTM. @MohamedAshiqrh PTAL.

**Pull request merged**